deployment/terraform/strings.go

```diff
@@ -294,13 +294,21 @@ net.ipv4.tcp_notsent_lowat = 16384
 
 # TCP buffer sizes are tuned for 10Gbit/s bandwidth and 0.5ms RTT (as measured intra EC2 cluster).
 # This gives a BDP (bandwidth-delay-product) of 625000 bytes.
-net.ipv4.tcp_rmem = 4096 156250 625000
-net.ipv4.tcp_wmem = 4096 156250 625000
-net.core.rmem_max = 312500
-net.core.wmem_max = 312500
-net.core.rmem_default = 312500
-net.core.wmem_default = 312500
```

Comment:
These were not needed since the second value in `tcp_rmem` and `tcp_wmem` overrides them.